packages/devtools-node/src/fs.ts

```diff
@@ -36,12 +40,12 @@ export async function createComposite(ceramic: CeramicClient, path: PathInput):
 export async function readEncodedComposite(
   ceramic: CeramicClient | string,
   path: PathInput,
-  index?: boolean,
+  deploy: boolean,
```

Comment:
This is misleading, reading from JSON doesn't deploy the models to the node, please keep as-is.

Reply:
It doesn't though. Composite create is the only command that creates (deploys) models to the node. `readEncodedComposite` deploys in the same manner that the deploy command deploys. For consistency between commands and SDK, everything that deploys (indexes) now has a deploy parameter.